doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -388,7 +388,7 @@ Protocol serialization samples
 SDFW samples
 ------------
 
-|no_changes_yet_note|
+* Removed the ``ssf_client`` sample since all services being shown has been removed.
```

Comment:
```suggestion
* Removed the ``ssf_client`` sample, as all services demonstrated by the sample have been removed.
```

Reply:
I also have a comment regarding this entry. I thought it was implemented already, but no. ssf_client need to be opened and written as SDFW: Service Framework Client

Reply:
I joined your suggestions